### PR TITLE
Change Cartoon Network (720p) to Cartoon Network (1080p)

### PR DESCRIPTION
**Pull request by kirigizakazutovn**

### CODE REVIEW

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
if the first link still works, why deleting it ? you can keep it as a secondary link.

to be put on vn_fptplay.m3u file.

adaptive is https://vips-livecdn.fptplay.net/hda3/cartoonnetworkhd_vhls.smil/playlist.m3u8

File: streams/us.m3u [hunk 1]
```diff
@@ -250,8 +250,8 @@ https://tve-live-lln.warnermediacdn.com/hls/live/2023167/tooneast/slate/VIDEO_2_
 http://70.39.83.58:8278/cartoonnetworkhindikdin/playlist.m3u8
 #EXTINF:-1 tvg-id="CartoonNetworkLatinAmerica.us" status="timeout",Cartoon Network Latin America
 http://209.91.213.10:8088/play/a00k
-#EXTINF:-1 tvg-id="CartoonNetworkVietnam.vn" status="error",Cartoon Network Vietnam (720p) [Geo-blocked]
-https://htv-drm-live-cdn.fptplay.net/CDN-FPT02/CARTOON-SD-720p/playlist.m3u8
+#EXTINF:-1 tvg-id="CartoonNetworkVietnam.vn" status="online",Cartoon Network Vietnam (1080p) [Geo-blocked]
+https://vips-livecdn.fptplay.net/hda3/cartoonnetworkhd_vhls.smil/chunklist_b5000000.m3u8
```
Comment (freearhey) [inline streams/us.m3u:254]: image

**Comment by kirigizakazutovn** [comment]
Screenshot_2022-06-20-14-13-32-21
Screenshot_2022-06-20-14-09-05-09

**Comment by kirigizakazutovn** [comment]
> 
The first link provided by HTVC through FPT Play But because HTVC has stopped cooperating with FPT Play, it doesn't work now There is a stream link hosted on Viettel's server 
[they are very strict about security, so can't get the link easily]

**Comment by kirigizakazutovn** [comment]
In general, pay TV in Vietnam is also quite cheap About 3.79 USD for FPT Play MAX package
 For easier understanding: With a bowl of pho No. 10 Ly Quoc Su of Mr. Tan 1 Cu

**Pull request closed without merging**